web-ng/src/app/components/layer-list/layer-list.component.ts

```diff
@@ -41,4 +45,8 @@ export class LayerListComponent {
         )
       );
   }
+
+  onAddLayer() {
+    this.routerService.setLayerId(':new');
```

Comment:
Please define':new' as a static constant in RouterService and reference it here.